### PR TITLE
Docs

### LINKED ISSUE #8: Re-knit README.Rmd
:-)

### PR DESCRIPTION
**Pull request by maelle**
* I have added myself and the Authors@R syntax to DESCRIPTION.

* I haven't added a vignette, I've added details to the docs of  `whoami` function instead.

* I added an item to NEWS about the Windows "bug" fix.

* This does *not* fix #8

### CODE REVIEW

File: R/whoami.R [hunk 1]
```diff
@@ -308,6 +308,19 @@ lookup_gh_username <- function(email, token) {
...
 #' whoami()
 #' }
+#' @details 
+#' For the username it tries the `LOGNAME`, `USER`, `LNAME` and
+#' `USERNAME` environment variables first. If these are all unset,
+#' or set to an empty string, then it tries running `id` on Unix-like
+#' systems and `whoami` on Windows.
+#' 
+#' For the full name of the user, it queries the system services and
+#' also tries the user's global git configuration. On Windows, it tries finding the global git configuration in `Sys.getenv("USERPROFILE")` if it doesn't find it in `Sys.getenv("HOME")` (often "Documents").
```
Comment (gaborcsardi) [inline R/whoami.R]: Can you pls break long lines? Thanks.

Comment (maelle) [reply R/whoami.R]: done but maybe a bit too much now 🤔

**Comment by gaborcsardi** [comment]
Yeah, I usually use max  75 characters.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**